rosidl_typesupport_cpp/resource/srv__type_support.cpp.em

```diff
@@ -0,0 +1,102 @@
+// generated from rosidl_typesupport_cpp/resource/msg__type_support.cpp.em
```

Comment:
copy paste, this should be `// generated from rosidl_typesupport_cpp/resource/srv__type_support.cpp.em`